workspace-loader/webpack.dev.js

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2015-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ *******************************************************************************/
+
+const merge = require('webpack-merge');
+const common = require('./webpack.common.js');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = merge(common, {
+    devtool: 'inline-source-map',
+    module:{
+        rules:[
+            {
+                test: /\.css$/,
+                use: [{
+                    loader: "style-loader" // creates style nodes from JS strings
+                }, {
+                    loader: "css-loader" // translates CSS into CommonJS
+                }]
+            }
+        ]
+    },
+    devServer: {
+        contentBase: './dist',
+        port: 3050,
```

Comment:
any reason to use 3050 instead of classical 3000 ?

Reply:
Just a historical reason. Changed to 3000.